scripts/run_miri_suite.sh

```
@@ -0,0 +1,16 @@
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$ROOT_DIR"
+
+HARDENING_TOOLCHAIN="${HARDENING_TOOLCHAIN:-nightly-2025-07-14}"
+export MIRIFLAGS="${MIRIFLAGS:-"-Zmiri-strict-provenance -Zmiri-symbolic-alignment-check -Zmiri-disable-isolation"}"
+
+source "$ROOT_DIR/scripts/hardening_test_names.sh"
+
+cargo +"${HARDENING_TOOLCHAIN}" miri setup
+
+for test_filter in "${hardening_test_filters[@]}"; do
```

Shellcheck (0.11.0)

[warning] hardening_test_filters is referenced but not assigned.

(SC2154)